ORStools/gui/directions_gui.py

```diff
@@ -31,11 +31,44 @@
 
 from PyQt5.QtWidgets import QCheckBox
 
-from ORStools.utils import transform
+from ORStools.utils import transform, exceptions
+
+
+def _get_avoid_polygons(layer):
```

Comment:
why is this an internal function and not a method of `Directions` as before?

Reply:
This was moved to class method of `Directions` again as it needed the parent `self.dlg`.